experiments/functional/scale_jiva_replica/test.yml

```diff
@@ -42,7 +42,7 @@
          register: replica_node_name
 
        - name: Getting the node name where replica is not scheduled
-         shell: kubectl get nodes -l {{ node_label }} -o jsonpath='{.items[?(@.metadata.name!="{{ replica_node_name.stdout }}")].metadata.name}' | tr " " "\n"
+         shell: kubectl get nodes --no-headers | grep -v master | awk {'print $1'}
```

Comment:
@w3aman remove only the node label in this task don't change the command the get the node_name

Reply:
Done